sdk/cognitiveservices/cognitiveservices-contentmoderator/src/operations/imageModeration.ts

```diff
@@ -294,21 +294,21 @@ export class ImageModeration {
 
   /**
    * Returns probabilities of the image containing racy or adult content.
-   * @param contentType The content type.
-   * @param imageUrl The image url.
+   * @param contentType The content type returned. Example: 'application/json'
```

Comment:
Rather this is the content type of the payload being sent to the server, which is 'application/json' for the case where you want to send a URL: https://westus.dev.cognitive.microsoft.com/docs/services/57cf753a3f9b070c105bd2c1/operations/57cf753a3f9b070868a1f66c

This is poor codegen though, since you'd never use this overload with a different content type, meaning we could have omitted the content type entirely from the signature.

Reply:
Updated text